Snake/Properties.java

```diff
@@ -82,33 +82,33 @@ static Theme getTheme() {
         return theme;
     }
     
-    static void Rainbow () {
+    static void rainbow () {
```

Comment:
You fixed the method names so that they match the java style, but the actual method name does not acurately describe what it does. 
Can you find a better name?